docs/source/installation/guide_complete/index.rst

```diff
@@ -0,0 +1,363 @@
+
+      .. button-ref:: installation:guide-complete:create-profile:core-sqlite-dos
+         :click-parent:
+         :expand:
+         :color: primary
+         :outline:
+
+         Create a ``core.sqlite_dos`` profile
+
+   .. grid-item-card:: :fa:`bolt;mr-1` ``core.psql_dos``
+      :text-align: center
+      :shadow: md
+
+      Use this for production work where database performance is important.
+
+      This storage plugin uses PostgreSQL for the database and provides the greatest performance.
+
+      +++++++++++++++++++++++++++++++++++++++++++++
+
+      .. button-ref:: installation:guide-complete:create-profile:core-psql-dos
+         :click-parent:
+         :expand:
+         :color: primary
+         :outline:
+
+         Create a ``core.psql_dos`` profile
+
+
+.. seealso::
+
+    See the :ref:`topic on storage <topics:storage>` to see a more detailed overview of the storage plugins provided by ``aiida-core`` with their strengths and weaknesses.
+
+Other packages may provide additional storage plugins, which are also installable through ``verdi profile setup``.
+
+
+.. _installation:guide-complete:create-profile:common-options:
+
+Common options
+--------------
+
+The exact options available for the ``verdi profile setup`` command depend on the selected storage plugin, but there are a number of common options and functionality:
+
+* ``--profile``: The name of the profile.
+* ``--set-as-default``: Whether the new profile should be defined as the new default.
+* ``--email``: Email for the default user that is created.
+* ``--first-name``: First name for the default user that is created.
+* ``--last-name``: Last name for the default user that is created.
+* ``--institution``: Institution for the default user that is created.
+* ``--non-interactive``: By default, the command prompts to specify a value for all options.
+  Alternatively, the ``--non-interactive`` flag can be specified, in which case the command never prompts and the necessary options need to be specified directly on the command line.
+  This is useful when ``verdi profile setup`` is used in non-interactive environments, such as scripts.
+* ``--config``: Instead of passing all options through command line options, their values can be defined in a YAML file and its filepath provided with this flag.
+
+In addition to these options, there are also a number of common options to configure the RabbitMQ broker which are described in more detail in the next section.
+
+.. _installation:guide-complete:create-profile:rabbitmq:
+
+RabbitMQ
+--------
+
+`RabbitMQ <https://www.rabbitmq.com/>`_ is a message broker that is used by AiiDA to scale by allowing to run multiple processes in the background through a daemon.
+The ``verdi profile setup`` command by default configures RabbitMQ as the broker and prompts for the connection parameters.
+If RabbitMQ is not available or the profile should be configured without a broker for another reason, run:
```

Comment:
With PR #6480 this logic has changed a bit, just make sure to update it in that PR